include/seqan3/alphabet/aminoacid/aa27.hpp

```diff
@@ -31,22 +31,37 @@
 // DAMAGE.
 //
 // ============================================================================
-// Author: Sara Hetzel <sara.hetzel AT fu-berlin.de>
-// ============================================================================
+
+/*!\file alphabet/nucleotide/aa27.hpp
+ * \ingroup aminoacid
+ * \author Sara Hetzel <sara.hetzel AT fu-berlin.de>
+ * \brief Contains seqan3::aa27, container aliases and string literals.
+ */
 
 #pragma once
 
 #include <cassert>
 
+#include <string>
+#include <vector>
+
 #include <seqan3/alphabet/concept.hpp>
```

Comment:
move this to l349 since it is not normally required, just for the test. Instead add here an include to core/platform.hpp as minimal include.